ceph-build-os/config/definitions/ceph-build-os.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: ceph-build-os
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && (ceph_build_xenial) && rebootable
+    display-name: 'ceph: Build OS'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 300
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+        - string:
+            name: DIST
+            description: "The distribution codename like xenial,centos,jessie"
+
+        - string:
+            name: DVER
+            description: "The distribution codename like C7.2, U16.04, D8"
+
+        - string:
+            name: VERSION
+            description: "The release of that particular build : 1.0.0, 1.1.0, ..."
+
+        - string:
+            name: REPOSITORY
+            description: "URL of the package repository, like http://debian.mirrors.ovh.net/debian/"
```

Comment:
It would be nice not to require a user to specify this all the time though. I understand that there is a preference depending on location and that is just not possible to compute, but it should have some defaults to that we aren't required to pass this in.

Can you also add the RPM repo url as an example to the help string?